libs/natparse/src/main/java/org/amshove/natparse/lexing/SyntaxKind.java

```diff
@@ -417,6 +418,7 @@
 	GLOBAL(true, false, false),
 	GLOBALS(true, false, false),
 	GREATER(true, false, false),
+	GROUP(true, false, false),
```

Comment:
The one test fails because `GROUP` has to be added to the [KeywordTable](https://github.com/MarkusAmshove/natls/blob/main/libs/natparse/src/main/java/org/amshove/natparse/lexing/KeywordTable.java#L245). Then the analyzer can see that `GROUP` is a keyword and raise the diagnostic if it is used as an identifier.